README.md

```diff
@@ -342,6 +343,28 @@ You can also set the annoation on existing Pods with this one-liner:
 ```bash
 $ kubectl get po --all-namespaces -o json | jq -r '.items[] | select(.spec.volumes[].hostPath or .spec.volumes[].emptyDir) | [ .metadata.namespace, .metadata.name ] | @tsv' | while IFS=$'\t' read -r namespace pod; do echo "\n NAMESPACE: $namespace \n POD: $pod \n"; kubectl annotate pod -n $namespace $pod "cluster-autoscaler.kubernetes.io/safe-to-evict=true"; done
 ```
+## Mutate Image Pull Policy
+
+There are cerntain images which require the enforcement of the ImagePullPolicy according to different user scenarios
+- IfNotPresent
+It can reduce the unnecessary traffic(Auth and Download requests) to Image repository and reuse the image which is cached on the node 
+- Always
+It can be useful when there require the absolute isolation in multi-tenant cluster, which prevents others to reuse the image cached on the node, for example: The image protected with ImagePullSecret from private repository is cached on the node after first successful pull, other use can be directly pull from node without proper auth
```

Comment:
Good explanation of the reason. I was wondering about the use case. I could not find any resources that the image is not put into the cache with `Always`. The [doc](https://kubernetes.io/docs/concepts/containers/images/) states that it is not pulled from there with config. Do you have some references?

Reply:
I think the logic here is that if the image is cached already, a malicious pod trying to use the image would get the ImagePullPolicy: Always, which would fail without the ImagePullSecret for its private registry.